code/__datastructures/stack.dm

```diff
@@ -0,0 +1,66 @@
+/datum/stack
```

Comment:
We already have this under `/Stack`. Remove one of the two.

Reply:
We also had a third one from 11 years ago, removed that one too, neither were even used